velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -760,13 +760,6 @@ PlanBuilder::AggregatesAndNames PlanBuilder::createAggregateExpressionsAndNames(
 
     agg.distinct = untypedExpr.distinct;
 
-    if (!untypedExpr.orderBy.empty()) {
```

Comment:
Would it be possible to keep this check by enhance it to apply only when aggregate function is order-sensitive?

Reply:
Done.